src/components/footer/footer.component.ts

```
@@ -5,22 +5,185 @@ import { style } from './footer.style';
+      );
+    }
+  }
+
+  /**
+   * Removes the cookie management script from the document.
+   * This method provides cleanup functionality to prevent memory leaks
+   * and ensures proper script lifecycle management.
+   * @private
+   */
+  private removeCookieScript(): void {
+    if (this.cookieScriptElement && this.cookieScriptElement.parentNode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)